### PR TITLE
Added govcloud domain support

### PR DESCRIPTION
**Pull request by vishaalcharpe**
Added govcloud domain support

### CODE REVIEW

File: script/fluentd.conf [hunk 1]
```diff
@@ -25,6 +25,7 @@
 <match pubsub.publish>
     @type lm
     company_name "${LM_COMPANY_NAME}"
+    company_domain "${LM_COMPANY_DOMAIN}"
```
Comment (ashishkucheria) [inline script/fluentd.conf:28]: Where is this used?

Comment (vishaalcharpe) [reply script/fluentd.conf:28]: The fluentd repository which is used by gcp to ingest logs.

**Review by ashishkucheria — APPROVED** [review]

**Comment by choudharypooja** [comment]
Please update the README to export this env variable just like others

**Comment by vishaalcharpe** [comment]
vm.sh is updated to have default domain as "logicmonitor.com" in case LM_COMPANY_DOMAIN is empty or null or " ".
README is also updated.
@choudharypooja please validate once again.

**Review by choudharypooja — APPROVED** [review]

**Pull request merged**